src/main/view/game/StructureOverviewPanel.java

```diff
@@ -1,33 +1,219 @@
-		drawPanelBox(g, screenDimensions);
-		Font oldFont = g.getFont();
-		g.setFont(getAssets().getFont(2));
-		g.fillText("Structure Overview", screenDimensions.x / 2 - 370,
-				screenDimensions.y / 2 - 245);
-		g.setFont(getAssets().getFont(1));
-		g.setFont(oldFont);
+		scrollPane.toFront();
+		updatePositions(screenDimensions);
+		updateStructures();
 	}
 
-	public void hideGUIElements() {
+	private void updatePositions(Point screenDimensions) {
+		scrollPane.setMaxWidth(screenDimensions.x - 148);
+		structureTable.setPrefWidth(screenDimensions.x - 148);
+		scrollPane.setMaxHeight(488);
+		scrollPane.setTranslateX(74);
+		scrollPane.setTranslateY(50);
+		structureTypeColumn.setPrefWidth(structureTable.getWidth()/7);
+		healthColumn.setPrefWidth(structureTable.getWidth()/7 - 13);
+		attackColumn.setPrefWidth(structureTable.getWidth()/7 - 13);
+		defenseColumn.setPrefWidth(structureTable.getWidth()/7 - 13);
+		armorColumn.setPrefWidth(structureTable.getWidth()/7 - 13);
+		upkeepColumn.setPrefWidth(structureTable.getWidth()/7 - 13);
+		workerColumn.setPrefWidth(structureTable.getWidth()/7 - 13);
+	}
+
+	private void updateStructures() {
+		ArrayList<Structure> structures = getAdapter().getStructures();
+        structureTable.setPrefHeight(45 + (structures.size() * 35));
+		structureList.clear();
+		for (Structure structure : structures) {
+			String structureType = "";
+			switch (structure.getType()) {
+			case CAPITOL:
+				structureType = "Capitol";
+				break;
+			case FARM:
+				structureType = "Farm";
+				break;
+			case MINE:
+				structureType = "Mine";
+				break;
+			case PLANT:
+				structureType = "Plant";
+				break;
+			case OBSERVE:
+				structureType = "Observation Tower";
+				break;
+			case FORT:
+				structureType = "Fort";
+				break;
+			case UNIVERSITY:
+				structureType = "University";
+				break;
+			default:
+				break;
+			}
+			StructureStats stats = structure.getStats();
+			//Have no way to get attack or worker count right now, leaving at -999 until later
+			structureList.add(new StructureItem(structureType, stats.getHealth(), -999, stats.getDefPow(),
+					stats.getArmor(), (int)stats.getUpkeep(), -99));
+			}
 	}
 
 	public void showGUIElements() {
+		root.getChildren().add(scrollPane);
+	}
+
+	public void hideGUIElements() {
+		root.getChildren().remove(scrollPane);
+	}
+	
+	class StructureItem {
+		private final SimpleStringProperty structureTypeProp;
+		private final SimpleStringProperty attackProp;
+		private final SimpleStringProperty  healthProp;
+		private final SimpleStringProperty  defenseProp;
+		private final SimpleStringProperty  armorProp;
+		private final SimpleStringProperty  upkeepProp;
+		private final SimpleStringProperty  workerProp;
+		
+		public StructureItem(String structureType, int health, int attack, int defense, int armor, int upkeep, int workers) {
+			structureTypeProp = new SimpleStringProperty(structureType);
+			healthProp = new SimpleStringProperty(health + "");
```

Comment:
probably better to use Integer.toString(health). It can get weird sometimes if the first part of a concatenation is not a string.